src/api/config/block-sync-params.md

```diff
@@ -0,0 +1,27 @@
+# Block Sync Parameters
+
+Explain module
+
+## `block_sync.block_batch_amount`
+
+- **Type:** Number
+- **Default:** $4$
+
+The number of blocks that can be sent in one message.
+
+```toml
+[block_sync]
+block_batch_amount = 4
+```
+
+## `block_sync.gossip_period`
+
+- **Type:** String or Number, [Duration](glossary#type-duration)
+- **Default:** 10 seconds
```

Comment:
is it seconds <seconds> or <secs> as in the example below? What will actually be accepted by the `gossip_period` parameter?

Reply:
`gossip_period` accepts a **String** or a **Number**, and interprets it as a **Duration**. Duration is explained at the link.

For Duration types, I specify their "Default" values as a human readable string like "30 seconds" or "5 minutes".

In the examples, I specify any valid value. Here is a question: maybe, we should change example to show the default value for each config parameter?

Reply:
My last comment doesn't have a part of the message for some reason. I was actually interested in the terminology.
In the "Default" you have "10 seconds", while in the example you have "5 secs". 
So, the question is: does a peer have to write out "seconds" or "secs", or just a plain number in the "gossip_period" field? Juding by your reply, it seems like peers can input minutes as well.
For instance, if I want the interval to be 3 minutes and 22 seconds, should I input "3 mins, 22 secs" or "202 secs", or any other variation?
I'd like to make it clear for the readers

Reply:
I see, and it is why I put the link `[Duration](glossary#type-duration)`. That section explains all the possible inputs for durations.

Should it be pointed out more explicitly in some way?